ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -104,25 +111,25 @@ static int
 azure_customize_file_system(WT_STORAGE_SOURCE *storage_source, WT_SESSION *session,
   const char *bucket, const char *auth_token, const char *config, WT_FILE_SYSTEM **file_system)
 {
+    // Get the value of the config key from the string
```

Comment:
Missing full stop at the end of this sentence.

Reply:
Corrected